_posts/2011-12-04-global-code-retreat-2011-in-berlin.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+author: Matthias Günther
+authorname: Matthias Günther
+title: Global Code Retreat 2011 in Berlin
+preview: I attended the Global Coderetreat event initialized by Corey Haines2. The general purpose of this event was to get a bunch of hackers together to implement Conway’s Game of Life.
+---
+
+I attended the [**Global Coderetreat**](http://coderetreat.org/ "coderetreat") event initialized by [Corey Haines](http://coreyhaines.com/ "Corey Haines"). The general purpose of this event was to get a bunch of hackers together to implement [Conway’s Game of Life](http://en.wikipedia.org/wiki/Conway%27s_Game_of_Life "Conway's Game of Life").
+
+![](http://farm8.staticflickr.com/7029/6452198253_ca6e01b7f4.jpg "Coderetreat@Immobilienscout alt")
+
+
+## Setting up goals
+
+As **J. B. Rainsberger** mentioned in his article about [four elements of simple design](http://www.jbrains.ca/permalink/the-four-elements-of-simple-design "four elements of simple design"), it is best to setup goals for your retreat to get most out of it. Here is a list of what I achieved in every section:
+
+1.  Being a mentor of TDD: I taught [dedan](https://github.com/dedan "dedan") how to use TDD, why  
+    TDD helps you to think about your design
+2.  Being a mentor of BDD: Showing a fellow Rubyist how to do BDD
+3.  Doing something completely stupid: I played _evil coder_ and _ping-pong pairing_ with  
+    [@tilosoenke](http://twitter.com/#%21/tilosoenke "@tilosoenke") - we couldn’t communicate with  
```

Comment:
That link is broken (and the `alt` extra is redundant). Should be `https://twitter.com/tilosoenke`